core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -666,18 +666,6 @@ public InodeDirectoryView getRoot() {
         }
       }
     }
-    if ((pathIndex < (pathComponents.length - 1)
```

Comment:
This update is moved into `applyInode` so that all modification time updates are consistently made in `InodeTreePersistentState`

Reply:
never mind, it's more complicated since we don't always want to update the parent id (e.g. if we are loading an inode from ufs)